### PR TITLE
fix(MatrixBuilder): allow using typed array in setMatrix

### PR DESCRIPTION
**Pull request by agirault**

### CODE REVIEW

File: Sources/Common/Core/MatrixBuilder/index.js [hunk 1]
```diff
@@ -110,7 +110,13 @@ class Transform {
   }
 
   setMatrix(mat4x4) {
-    if (Array.isArray(mat4x4) && mat4x4.length === 16) {
+    if (!mat4x4) {
+      return this;
+    }
+    if (
+      Object.prototype.hasOwnProperty.call(mat4x4, 'length') &&
```
Comment (jourdain) [inline Sources/Common/Core/MatrixBuilder/index.js]: Do you really need that line? (117)

Comment (agirault) [inline Sources/Common/Core/MatrixBuilder/index.js]: @jourdain still returning false :thinking:

Comment (jourdain) [reply Sources/Common/Core/MatrixBuilder/index.js]: Hence my comment..

**Comment by jourdain** [comment]
LGTM

**Comment by jourdain** [comment]
Do you have other fix for today? Or should I merge it ?

**Comment by agirault** [comment]
no other fix so far but it's not urgent either.

**Comment by jourdain** [comment]
ok, will wait till the end of the day in case we want to tackle other things too...

**Pull request merged**